### ISSUE TITLE
tuttleofx python2 / python3 binding

### ISSUE THREAD

**Issue by fabiencastan** [cbenhagen/homebrew-video#8]
We need be able to build for python2 and/or python3.

Here is a good example:
https://github.com/Homebrew/homebrew/blob/master/Library/Formula/sip.rb

**Comment by fabiencastan** [comment]
Should looks like:
```
  depends_on :python => :recommended
  depends_on :python3 => :optional

  def install
    Language::Python.each_python(build) do |python, version|
      mkdir_p "build_release_py#{version}"
      cd "build_py#{version}"
      system "cmake", "..",
                   "-DCMAKE_INSTALL_PREFIX=#{prefix}",
                   "-DCMAKE_BUILD_TYPE=RELEASE"
                   "-DPYTHON_EXECUTABLE=#{python}"
      system "make", "install/strip"
    end
  end
```

**Comment by cbenhagen** [comment]
is this fixed in 970c3787175ad541a4af5a86a7df9d51456b9377?

**Comment by fabiencastan** [comment]
Yes
